templates/openstackclient/bin/tripleo-deploy.sh

```diff
@@ -49,12 +49,6 @@ play() {
     echo "Then re-run '$0 -p' to run the new playbooks."
     exit
   fi
-  if git branch | grep " tripleo_deploy_working$" > /dev/null; then
-    git checkout tripleo_deploy_working >/dev/null
-    git reset --hard refs/tags/latest
-  else
-    git checkout -b tripleo_deploy_working latest
-  fi
```

Comment:
I had this code here so that (on first execution) running tripleo-deploy.sh -p would run correctly without error. Also, my thought was the actual "checkout" belonged to -p since it would be actually using the local checkout anyway. So let me ask this: does tripleo-deploy.sh -p run cleanly (on a fresh git repo/install) now?

Reply:
right, had to add a check if /home/cloud-admin/playbooks/tripleo-ansible exists